src/Clapi/Attributor.hs

```diff
@@ -0,0 +1,26 @@
+module Clapi.Attributor where
+import Control.Monad (forever)
+
+import Clapi.Server (ClientEvent(ClientData), AddrWithUser(..))
+import Clapi.Protocol (Protocol, waitThen, sendFwd, sendRev)
+import Clapi.Types (Message(..))
+import Clapi.NamespaceTracker (Om, Ownership(Client))
+
+-- The attributee stuff needs to be lining up through everything less hokily
+attributeClientMsgs :: String -> [Om] -> [Om]
+attributeClientMsgs u oms = map attributeClientMsg oms
```

Comment:
This is all fine, just a thought: I tend to try to define just the singular function (the `String -> Om -> Om`) that you've defined in the `where` clause at the top level, and then use the lifting/mapping/composition operations where I need to used it. That would simply make a `attributor` do `attributeClientMsg u <$> ms` at the end of line 25.